sdk/ml/azure-ai-ml/setup.py

```diff
@@ -86,6 +86,8 @@
         "azure-common<2.0.0,>=1.1",
         "typing-extensions<5.0.0",
         "opencensus-ext-azure<2.0.0",
+        "opencensus-ext-logging",
```

Comment:
Should this have a minimum version pinned? I'm not sure if that's required or not.

Reply:
Not required, but it is good practice. I'll update that.